cli/smartcontract/smart_contract.go

```diff
@@ -92,6 +92,10 @@ func NewCommands() []cli.Command {
 						Name:  "debug, d",
 						Usage: "Debug mode will print out additional information after a compiling",
 					},
+					cli.StringFlag{
+						Name:  "emitdebug",
```

Comment:
Short form? And maybe it's better to rename `debug` flag into `verbose` or something like that and use `debug` here.

Reply:
https://github.com/nspcc-dev/neo-go/pull/829